docs/tutorials/redirect-fields-with-python.md

```diff
@@ -0,0 +1,92 @@
+# Redirect fields from responses with Python
+
+This tutorial will give a simple example how you can split complex data coming from a block into simpler structures, using the python block.
+For this we have to join a couple of inseri blocks.
+
+## Step 1: Create a new post
+
+On your inseri instance or in the playground (remember, your changes will not be saved there), add a new post and give it a title.
+In this example, we call the post "Image selection". You can type this at "Add title...".
+
+## Step 2: Add a Web API block
+
+Add an inseri text editor by clicking the `+` below the title.
+Select "Browse all" and look for "Web API" in the category _inseri_.
+
+Enter the following URL in the corresponding field:
+
+```
+https://raw.githubusercontent.com/inseri-swiss/inseri-core-wp/main/docs/assets/book_metadata.json
+```
+
+Choose the format "JSON" because that is what will be read by the dropdown that we are going to add later.
+
+Now, give your text editor a meaningful name.
+For this, open the configuration panel on the right and enter "webApi-book" under _block name_.
+
+## Step 3: Add a python block
+
+Add a "Python Code" block the same way you added the web API block.
+
+Once you have the block in your post, select "Write Code". This way you can add the code directly in this block.
+
+First, set the _block name_ to `python-divide` available in the additional parameters.
+
+Copy this snippet into the Python code editor:
+
+```python
+title = data["title"]
+author = data["author"]
+year = data["year"]
+publisher = data["publisher"]
+```
+
+Obviously, this is not yet complete because `data` is not defined anywhere in the code.
+
+## Step 4: Open the extended view for the python block
+
+Under "Inputs from Blocks", enter the variable name `data` and accept ith with selecting the `+`.
+You can now select `webApi-book: data` to connect this block to the web API block.
+
+Now, you can set several output channels.
+As with the input, enter a variable name from the script into the textbox and accept with `+`.
+For `title` and `author` set the content type to "Text".
```

Comment:
what about using json also for `title` and `author`. Not the most obvious choice but one can argue that is one of the most used content types so to get experience with it :D.

Reply:
I wanted to have something else than only json but maybe it's a bit too constructed this way. An option would be to re-write it to have one value in json and the other as text but the best would be an example where it matters.

Reply:
I don't have a strong opinion, it was only a remark.